packages/unity-bootstrap-theme/stories/molecules/card-and-image/card-and-image.templates.stories.js

```diff
@@ -66,16 +67,13 @@ export const LeftFloatedCard = createStory(args => {
       </div>
     </div>
   );
-});
-LeftFloatedCard.args = {
-  template: 1,
 };
 
-export const RightFloatedCard = createStory(args => {
+export const RightFloatedCard = ({left}) => {
```

Comment:
Should this be `right` ?

Reply:
`left` prop name is correct, it's a boolean set to false on line 112. 

`RightFloatedCard` and `LeftFloatedCard` Stories are very similar but they both use `left` as a control or prop to choose a class name on line 21 and 76.

But I think you might be correct that the prop name might make more sense, since the className is `uds-card-and-image-RIGHT`

I can change the `${left ? "" : "uds-card-and-image-right"}` to `${right && "uds-card-and-image-right"}`